src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -18,12 +20,16 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             int pageIndex,
             int resultCount,
             TimeSpan duration,
+            IEnumerable<TimeSpan> sourceTimings,
+            TimeSpan aggregationTime,
             LoadingStatus loadingStatus) : base("SearchPage")
         {
             base["ParentId"] = parentId.ToString();
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
+            base["SourceTimingDuration"] = string.Join(",", sourceTimings?.Select(e => e.TotalSeconds));
```

Comment:
`string.Join(",", null)` throws

Reply:
I don't think the null check is needed. 

I'll clean it up.